eos/signal-pdf.cc

```diff
@@ -19,22 +19,13 @@
  */
 
 #include <eos/signal-pdf.hh>
+#include <eos/signal-pdf-impl.hh>
 #include <eos/maths/power-of.hh>
-#include <eos/b-decays/b-to-d-l-x-nu.hh>
-#include <eos/b-decays/b-to-gamma-l-nu.hh>
-#include <eos/b-decays/b-to-psd-l-nu.hh>
-#include <eos/b-decays/b-to-pi-l-x-nu.hh>
-#include <eos/b-decays/b-to-pi-pi-l-nu.hh>
-#include <eos/b-decays/b-to-vec-l-nu.hh>
-#include <eos/b-decays/b-to-3l-nu.hh>
-#include <eos/b-decays/bs-to-kstar-l-nu.hh>
-#include <eos/b-decays/lambdab-to-lambdac-l-nu.hh>
-#include <eos/b-decays/lambdab-to-lambdac2625-l-nu.hh>
-#include <eos/rare-b-decays/b-to-k-ll.hh>
-#include <eos/rare-b-decays/b-to-kstar-ll.hh>
-#include <eos/rare-b-decays/b-to-psd-nu-nu.hh>
-#include <eos/rare-b-decays/b-to-vec-nu-nu.hh>
+#include <eos/b-decays/signal-pdfs.hh>
+#include <eos/rare-b-decays/signal-pdfs.hh>
 #include <eos/utils/density.hh>
+#include <eos/utils/instantiation_policy-impl.hh>
+#include <eos/utils/log.hh>
 #include <eos/utils/private_implementation_pattern-impl.hh>
 #include <eos/utils/wrapped_forward_iterator-impl.hh>
 
```

Comment:
Can you remove lines 55 - 56 below? In the function `norm`, the `using std::pow` is not needed.

Reply:
Done.